src/Tribe/Tickets/Attendees_Table.php

```diff
@@ -14,9 +14,18 @@ class Tribe__Events__Tickets__Attendees_Table extends WP_List_Table {
 
 	/**
 	 * Class constructor
+	 *
+	 * @param array $args  additional arguments/overrides
+	 *
+	 * @see WP_List_Table::__construct()
 	 */
-	function __construct() {
-		parent::__construct( array( 'singular' => 'attendee', 'plural' => 'attendees', 'ajax' => true ) );
+	function __construct( $args = array() ) {
+		$args = wp_parse_args( $args, array(
```

Comment:
:+1: Yeah, I like that idea. Could you filter the args after the `wp_parse_args` call with a filter name of: `tribe_events_tickets_attendees_table_args'?